public/src/components/channelManagement/richTextEditor/richTextEditor.tsx

```diff
@@ -239,34 +236,36 @@ const FloatingLinkToolbar = () => {
   } = useFloatingLinkState();
   const active = useActive();
   const activeLink = active.link();
-  const { empty } = useCurrentSelection();
-  const linkEditItems: ToolbarItemUnion[] = useMemo(
-    () => [
-      {
-        type: ComponentItem.ToolbarGroup,
-        label: 'Link',
-        items: activeLink
-          ? [
-              { type: ComponentItem.ToolbarButton, onClick: () => clickEdit(), label: 'Edit link' },
-              { type: ComponentItem.ToolbarButton, onClick: onRemove, label: 'Remove link' },
-            ]
-          : [{ type: ComponentItem.ToolbarButton, onClick: () => clickEdit(), label: 'Add link' }],
-      },
-    ],
-    [clickEdit, onRemove, activeLink],
-  );
-
-  const items: ToolbarItemUnion[] = useMemo(() => linkEditItems, [linkEditItems]);
-
   return (
     <>
-      <FloatingToolbar items={items} positioner="selection" placement="top" enabled={!isEditing} />
-      <FloatingToolbar
-        items={linkEditItems}
-        positioner={linkPositioner}
-        placement="bottom"
-        enabled={!isEditing && empty}
-      />
+      <FloatingToolbar placement="top">
+        <CommandButtonGroup>
+          {activeLink ? (
+            <>
+              <button
+                className={`remirror-button ${active.bold() && 'remirror-button-active'}`}
```

Comment:
this is my fault - but this can just be:
`className={'remirror-button'}`
I think this was copied from the bold button!

Reply:
Removed, good spot!